core/src/main/java/org/apache/carbondata/core/datamap/DataMapStoreManager.java

```diff
@@ -126,20 +133,20 @@ private TableDataMap getTableDataMap(String dataMapName,
   /**
    * Clear the datamap/datamaps of a mentioned datamap name and table from memory
```

Comment:
please modify the comment also, datamap name is removed

Reply:
ok